### ISSUE TITLE
trouble building a stand alone toolchian

### ISSUE THREAD

**Issue by Char-Aznable** [easybuilders/easybuild#502]
I'm new to easybuild. I'm trying to build `eb foss-2019a.eb --robot` right after installing easybuild. However, I hit https://github.com/easybuilders/easybuild-easyblocks/issues/1350 where I'm stuck. If I understand it correctly, the tools in foss-2019a built by easybuild depends on the libraries provided by the OS -- is this the case? If not, what is the way to build a stand alone toolchain, including gcc, binutils, glibc, ...?

**Comment by akesandgren** [comment]
Exactly what do you mean by getting stuck?

The above (#1350) is not a problem, it's the way things are supposed to be.

EB builds a small set of packages that depends on the system compiler and libraries, which are then used to build the GCCcore compiler, the rest does not depend on system libraries
(Well libc is not built by EB)

**Comment by boegel** [comment]
@Char-Aznable Can you clarify which problem/errors you are hitting exactly?

**Comment by Char-Aznable** [comment]
I got the error when building foss-2019a:
```
build failed (first 300 chars): zlib is not statically linked in /home/aznb/easybuild/software/binutils/2.31.1-GCCcore-8.2.0/bin/addr2line
```

**Comment by akesandgren** [comment]
Can you upload the complete build log somewhere?
And just building binutils-2.31.1-GCCcore-8.2.0.eb should be enough to show the problem.

**Comment by boegel** [comment]
@Char-Aznable https://github.com/easybuilders/easybuild-easyblocks/issues/1350 was fixed last May (in EasyBuild v3.9.1).

Please try again with a more recent version of EasyBuild. If the problem persists, please let us know
